homeassistant/config.py

```diff
@@ -453,3 +455,31 @@ def merge_packages_customize(core_customize, packages):
         conf = schema(pkg)
         cust.extend(conf.get(CONF_CORE, {}).get(CONF_CUSTOMIZE, []))
     return cust
+
+
+@asyncio.coroutine
+def async_check_ha_config_file(hass):
+    """Check if HA config file valid.
+
+    This method is a coroutine.
+    """
+    import homeassistant.components.persistent_notification as pn
```

Comment:
I think it will become the same if I import this or I need to do: `from ...persistent_notification Import DOMAIN, SERVICE_XY` for calling the service later.